toc.json

```
@@ -343,6 +343,25 @@
       "title": "CRM APIs",
       "uri": "openapi/crm/crm.json"
     },
+
+    {
+      "type": "divider",
+      "title": "Vendasta APIs"
+    },
+    {
+      "type": "item",
+      "title": "Internal APIs",
```

```suggestion
      "title": "Vendasta APIs",
```
Am I right? crct me if I am wrong

---

No, you are right 👍